config/environments/production.rb

```diff
@@ -90,8 +86,6 @@
     host: Figaro.env.domain_name,
     protocol: 'https'
   }
-  config.action_mailer.delivery_method = Figaro.env.pt_mode == 'on' ? :letter_opener_web : :smtp
-  config.action_mailer.perform_deliveries = true
```

Comment:
No, that's probably fine as we'll most likely always use the default settings in production.